### ISSUE TITLE
Missing Function in Meter Class

### ISSUE THREAD

**Issue by emansih** [stripe/stripe-java#1852]
### Describe the bug

Hi!

I am using the Meter class(com.stripe.model.billing.Meter) and there is an inconsistency with the docs and the SDK itself. The docs mentioned there is `Meter.retrieve("").getEVentSummaries()`, however, I can't find the function.(https://docs.stripe.com/api/billing/meter-event_summary/list) 

### To Reproduce

Here is an example code snippet. 

```
        val meter = Meter.retrieve("METER_ID")
        val params = MeterEventSummaryListParams.builder()
            .setCustomer("CUSTOMER_ID")
            .setStartTime(1724289091L)
            .setEndTime(1724287891L)
            .build()

        val meterEventSummaries = meter.getEventSummaries().list(params)
                                      // ^^^ This fails to compile
```

`getEventSummaries()` is an unresolved reference. 

### Expected behavior

Expose getEventSummaries in the SDK. 

### OS

Windows

### Java version

17

### stripe-java version

26.7.0

### API version

2024-06-20

### Additional context

I will use the REST API endpoint as a workaround

**Comment by ramya-stripe** [comment]
Thanks for reporting @emansih 

Looks like the code snippet might be wrong...

Can you try `MeterEventSummary.list(meterId, params)`?

**Comment by emansih** [comment]
```
MeterEventSummary.list(meterId, params)
```
this works. the docs is wrong.

**Comment by ramya-stripe** [comment]
Thanks for confirming - we will look at fixing the docs

**Comment by ramya-stripe** [comment]
We will be adding the `getEVentSummaries()` on the Meter class in our next release this Thursday

**Comment by emansih** [comment]
ok. what is the difference between 

`MeterEventSummary.list(meterId, params)`


and 

`getEVentSummaries()`

**Comment by ramya-stripe** [comment]
They both call the same underlying API under the hood. It is just a matter of convenience.

**Comment by ramya-stripe** [comment]
This should now be fixed with the latest Java SDK version: 26.9.0
